tools/docker/lib/common

```diff
@@ -1,18 +1,15 @@
 #!/bin/bash
-set -ex
+set -euxo pipefail
 
 # Current supported architectures
 export ALL_TARGET_ARCH=(amd64 arm32v6 arm64v8)
 
-# Architecture used in tags with no architecture specified (certbot/certbot:latest, certbot/certbot:v0.35.0, ...)
-export DEFAULT_ARCH=amd64
-
 # Name of the Certbot Docker organizaation on GitHub. After creating
 # repositories with the same names (e.g. "certbot", "dns-dnsmadeeasy", etc.)
 # using a different account on Docker Hub, you can change this value to have
 # the scripts modify those Docker repositories rather than the repositories for
 # the official Certbot Docker images.
-export DOCKER_HUB_ORG="certbot"
+export DOCKER_HUB_ORG="humanoid2050"
```

Comment:
```suggestion
export DOCKER_HUB_ORG="certbot"
```

Reply:
knew i was going to make that mistake one of these days